src/imgui/imgui_widgets.cpp

```diff
@@ -6294,9 +6294,9 @@ bool ImGui::ColorButton(const char* desc_id, const ImVec4& col, ImGuiColorEditFl
             RenderFrameBorder(bb.Min, bb.Max, rounding);
         else
         #ifdef __APPLE__
-           window->DrawList->AddRect(bb.Min - ImVec2(3, 3), bb.Max + ImVec2(3, 3), GetColorU32(ImGuiCol_FrameBg), rounding * 2,NULL,4.0f);; // Color button are often in need of some sort of border
+           window->DrawList->AddRect(bb.Min - ImVec2(3, 3), bb.Max + ImVec2(3, 3), GetColorU32(ImGuiCol_FrameBg), rounding * 2,0,4.0f);; // Color button are often in need of some sort of border
```

Comment:
This is actually an error in clang on FreeBSD where NULL is a pointer, not an integer 0.